src/python/WMQuality/TestInit.py

```diff
@@ -185,23 +185,29 @@ def setSchema(self, customModules = [], useDefault = True, params = None):
         if not useDefault:
             defaultModules = []
 
-        # filter out unique modules
-        modules = {}
+        # filter for unique modules
+        seen = set()
```

Comment:
We don't actually use this `seen` variable. Same comment for below block.

Reply:
What do you mean, that what I use to filter.

Reply:
You use `seen` and `modules`, but only `modules` is really used. Thus `seen` can go away and you just update 
`if module not in seen:`
by
`if module not in modules:`

Reply:
Ok, guess I did my copy&paste from the internet too quickly ;-). Will fix and update the PR.